comprehensive-getting-started/Layer_Comprehensive_Getting_Started.ipynb

```diff
@@ -0,0 +1,868 @@
+      "metadata": {
+        "id": "bOKbz9uuakNQ"
+      },
+      "outputs": [],
+      "source": [
+        "# Step I: Install Layer and Authentication\n",
+        "\n",
+        "# Run the shell command to install Layer python package\n",
+        "!pip install layer -U\n",
+        "# Import Layer package\n",
+        "import layer\n",
+        "# Call Layer's login function to authenticate\n",
+        "layer.login()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "# **Step II:** Initialise a project and import some Layer packages\n",
+        "---\n",
+        "\n",
+        "Once you paste the authentication code into the Python prompt on your notebook, you will successfully log into Layer! Click on the link https://app.layer.ai and you will see the landing page of Layer.\n",
+        "\n",
+        "  <img src=\"  https://github.com/layerai/examples/blob/layer-comprehensive-demo/comprehensive-getting-started/images/Screenshot%202022-07-07%20at%2018.56.52.png?raw=true\" width=\"700\" height=\"400\" />\n",
+        "\n",
+        "Click on **My Projects** tab on the top bar. Once you create several projects using Layer, they will all list here. Click on the **New project** button and you will see a pop-up page to enter some details of your project.\n",
+        "\n",
+        "  <img src=\"  https://github.com/layerai/examples/blob/layer-comprehensive-demo/comprehensive-getting-started/images/Screenshot%202022-07-07%20at%2019.08.51.png?raw=true\" width=\"700\" height=\"400\" />\n",
+        "\n",
+        "  Once you create your project, you should see the empty project page of your project named **my-first-project** as seen in the image below.\n",
+        "\n",
+        "  <img src=\"  https://github.com/layerai/examples/blob/layer-comprehensive-demo/comprehensive-getting-started/images/Screenshot%202022-07-14%20at%2010.59.27.png?raw=true::\" width=\"700\" height=\"400\" />\n",
+        "\n",
+        "---\n"
+      ],
+      "metadata": {
+        "id": "M5Qm4gGMtJv5"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "### Layer Decorators or Functions Used In Step-II\n",
+        "---\n",
+        "\n",
+        "**layer.init():**<br>\n",
+        "To start logging into the ***my-first-project*** project, layer's init function should be called with the name of project as shown in the first line below. It basically tells Layer that everything will be saved in that specific project. If any project exists with the given name, then it will match and sync otherwise, a new project with the given name will be created under your organisation or personal account. This function also has a ***pip_packages*** parameter which takes a list of python packages. You are expected to list all Layer packages you use in your project. Layer will install those python packages first to make your project environment ready on the Layer Infra. <br><br>\n",
```

Comment:
Layer's init

Reply:
Python packages

Reply:
`You are expected to list all Layer packages you use in your project` is not entirely true, we do have some packages already installed. They should only specify the pages if the versions we have are different from the ones they want to use https://docs.app.layer.ai/docs/reference/fabrics#preinstalled-libraries

Reply:
If this is for demonstration purposes, we should add the package versions.

Reply:
Done